.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -57,21 +69,54 @@ assignees: ''
 
 ---
 
-- [ ] **Tag release on Github and attach `seqan-[VERSION]-with-submodules.tar.gz` to the release.**
+- [ ] **Tag release on Github and attach `seqan3-[VERSION]-{Linux,Source}.tar.xz{,.sha265}` to the release.**
+
+  <details><summary>Click for detailed steps</summary><br>
+
+  Github is not able to create annotated releases (https://github.com/seqan/product_backlog/issues/159), so we have to manually sign the release.
+  ```bash
+  git checkout release-3.x.y
+  git tag -s 3.x.y
+  git push upstream 3.x.y
```

Comment:
```suggestion
  git push upstream 3.x.y
```
What about `git push --tags`?

Reply:
that would push all tags, but you want to push only one tag (you might push other tags accidentally)